### PR TITLE
Update Release note to add stable version 4.4.0

### PR DESCRIPTION
**Pull request by tferhat-outpayce**

### CODE REVIEW

**Comment by coliva** [comment]
I would have expected to see some changes in the user guide.

**Comment by tferhat-outpayce** [comment]
Just added the changes on the readme

File: userguide.md [hunk 1]
```diff
@@ -63,16 +63,16 @@ With these very few steps you can collect payments in your web application:
 
 ### Step 1: Integration
 
-Include the following scripts and CSS in your web page.
+Include the following scripts and CSS in your web page, replacing `<sdk_version>` by the version of the SDK that you want to use. Check Releases (#releases) section for available versions.
 
 ```HTML
-<script src="https://prd.payment.amadeus.com/checkout/sdk/4.1.1/sdk-es2015.js" integrity="sha384-gtU0sxD2E+UyHcDYNIsHzpgMxb9Ah5b/JIyC45Z9jrRY7W375uzsr+XVf+ynxU9g" crossorigin="anonymous"></script>
-<link rel="stylesheet"  href="https://prd.payment.amadeus.com/checkout/sdk/4.1.1/sdk.css" integrity="sha384-PxY0l2o4ZNHU96cDetNVSb5wJDrxX63KNcT60xxnPjxvx4/uxfFxysATJleJHaaQ" crossorigin="anonymous">
+<script src="https://prd.payment.amadeus.com/checkout/sdk/<sdk_version>/sdk-es2015.js" integrity="sha384-<js_hecksum>" crossorigin="anonymous"></script>
```
Comment (coliva) [inline userguide.md]: Please add the correct version/checksum here.

Comment (tferhat-outpayce) [reply userguide.md]: I guess we need to wait the load to PROD to do that

File: release-notes.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
 # Outpayce Checkout SDK - Release Notes
 
+### 4.4.0 - Supported until 15 May 2025
+- Added the ability for users to customize URLs for better connectivity and flexibility with the checkout system.
```
Comment (pardele) [inline release-notes.md]: This functionality is only available in the Core SDK, so I wouldn't mention it here

File: release-notes.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+- Added New Payment Method:
+  - South Korean Payment Methods
+- New Logos Added: 
+  - Payment Methods: Airtel, MTN, Paga, T Money, Tigo Cash, UBA, Vodafone.
+- Added `milesAccountDetails` option to hide the frequent flyer details on miles-component
```
Comment (pardele) [inline release-notes.md]: Version 4.4.0 contains only the intranet, Korean payment method and the logos, no?
why do we have miles and split payment listed here?

File: userguide.md [hunk 4]
```diff
@@ -134,16 +133,18 @@ __`AmadeusCheckoutOptions` properties:__
...
 |`collectBillingAddressForCountries`|Array\[String\]|`[]`| List of country codes for which billing address is collected. If the option is not provided, billing address can be collected for any country.|
 |`forceChangeDetection`| `Boolean` |`false`| /!\ Not to be used in a standard use case /!\ In case your framework is messing up with application lifecycle, use this option to force change detection on CVV and PAN field. |
 |`emitCardFormDetails`|Boolean|`false`|If set to `true`, the onReady and onChange callbacks will be enriched with card form details information. See `cardForm` key in below documentation|
 |`emitContactFormDetails`|Boolean|`false`|If set to `true`, the onReady and onChange callbacks will be enriched with contact form details information. See `contactForm` key in below documentation|
 |`paymentMethodSpecificOptions.applePay`| `AmadeusCheckoutApplePayOptions` |`undefined`| Apple Pay specific options. See below for more details. |
+|`paymentMethodSpecificOptions.googlePay`| `AmadeusCheckoutGooglePayOptions` |`undefined`| Google Pay specific options. See below for more details. |
```
Comment (pardele) [inline userguide.md]: why is Google pay option here, if Google Pay is not in the release?

File: userguide.md [hunk 5]
```diff
@@ -169,6 +170,25 @@ options.paymentMethodSpecificOptions.applePay = {
 };
 ```
 
+__`AmadeusCheckoutGooglePayOptions` properties:__
```
Comment (pardele) [inline userguide.md]: why is Google pay option here, if Google Pay is not in the release?

File: userguide.md [hunk 6]
```diff
@@ -224,6 +244,7 @@ Parameters:
   - `type`: String. Type of payment method (eg. `AmopPaypal`). Used as a unique identifier for each payment method type (see table in appendix).
   - `displayableName`: String. Displayable name, only defined for alternative payment methods (eg. `PayPal`).
   - `hasFees`: Boolean. Indicate whether OB fee is applicable or not.
+  -`isConfigValid`:Boolean. Indicate wether the payment method configuration is valid or not.
```
Comment (pardele) [inline userguide.md]: typo here: whether

File: userguide.md [hunk 7]
```diff
@@ -475,21 +499,30 @@ Parameters:
 Please note that to avoid too many server calls (for obfees), a debounce timer of 300ms is present on the selectMopComponent(s) functions.
 Extra focus should still be put on the frequency this method is called to avoid unecessary computations.
 
-For a payment that includes QBRPoints, you can call this helper function to pass the QBRPoints informations:
+For a payment that includes QBRPoints or miles without slider UI, you can call this helper function to pass the loyalty informations:
```
Comment (pardele) [inline userguide.md]: I don't think that miles without slider is already released

File: userguide.md [hunk 8]
```diff
@@ -475,21 +499,30 @@ Parameters:
...
-- `qbrLoyalty`: Object containing the following properties:
-    - `accountNumber`: String.  QBRLoyalty Account Number.
-    - `program`: String. QBRLoyalty Program.
-    - `numberOfPoints`: String. Number of QBRPoints used for payment.
-    - `convertedAmount`: String. Equivalent amount of selected QBRPoints.
-    - `currency`: String. Type of currency for payment.
-    - `membershipId`: String. It is optional field and contains the frequent flyer number or the membership Identifier.
-    - `sliderRanks`: Number. Contains the maximum rank.
-    - `sliderSelection`: Number. Contains the specific rank for slider.
+- `loyaltyData`: Object containing either `qbrLoyalty` or `milesLoyalty`
```
Comment (pardele) [inline userguide.md]: I don' t think this change is needed either

**Review by nicolas-gex — APPROVED** [review]

**Pull request merged**